webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/BreadcrumbSwitcherPopover.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover, ListGroup, ListGroupItem, Pager, Icon } from 'patternfly-react';
+import EllipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import './BreadcrumbSwitcherPopover.scss';
+
+const BreadcrumbSwitcherPopover = ({
+  resources,
+  onNextPageClick,
+  onPrevPageClick,
+  loading,
+  hasError,
+  currentPage,
+  totalPages,
+  ...props
+}) => {
+  let popoverBody;
+
+  if (loading) {
+    popoverBody = (
+      <div className="breadcrumb-switcher-popover-loading text-center">
+        <Icon name="spinner" spin size="lg" />
+      </div>
+    );
+  } else if (hasError) {
+    popoverBody = (<div className="breadcrumb-switcher-popover-error">{__('Error: Unable to load resources.')}</div>);
```

Comment:
nitpick: doesnt look as great, maybe we can consider using the same messagebox we have in statistics that handle failure with a small icon etc.